pkg/cloudprovider/providers/openstack/openstack.go

```diff
@@ -85,6 +85,8 @@ type LoadBalancerOpts struct {
 	MonitorMaxRetries    uint       `gcfg:"monitor-max-retries"`
 	ManageSecurityGroups bool       `gcfg:"manage-security-groups"`
 	NodeSecurityGroupID  string     `gcfg:"node-security-group"`
+
+	EnableAnnotationFloatingNetworkId bool `gcfg:"enable-annotation-floating-network-id"` // allow override of floatingnetworkid in service
```

Comment:
@zetaab - We don't really need this config option, we should have the code enabled all the time.